scala_proto/scala_proto.bzl

```diff
@@ -308,12 +308,13 @@ def scala_proto_repositories():
     )
 
 def _colon_paths(data):
-  return ':'.join([f.path for f in data])
+  return ':'.join(["{root},{path}".format(root=f.owner.workspace_root, path=f.path) for f in data])
```

Comment:
this is making me nervous, but maybe that's the only way to go here.

There was an old doc about fixing these kinds of issues about cross workspace dependencies like 1.5 years ago or so, but I don't think they actually ever did anything about it, unless I'm mistaken:

cc @damienmg  I recall there was a plan to put all the paths rooted with the project name. That never shipped right?

Reply:
If there's a better way to deal with this, I would be happy to do that instead. Frankly I just cribbed this from what the `java_proto_library` tool is doing